webdev/phase-2.md

```diff
@@ -0,0 +1,124 @@
+  * [HTML  Foundations](https://www.theodinproject.com/paths/foundations/courses/foundations#html-foundations)
+
+## Module 2: Deep dive into HTML, CSS
+
+### Duration
+4 weeks
+
+### Facilitation
+1-on-1 Mentorship sessions
+
+### Learning Topics
+* How to run a local web server to start working on your static HTML page 
+* Structure of HTML page 
+* Adding metadata to your page
+* Style your HTML with CSS
+* Using CSS selectors to create classes and reusable styles 
+* CSS style hierarchy and overrides 
+* Embedding media with HTML
+* Adding web fonts to your page 
+
+### Key Mentee Deliverables 
+* Run a local web server to see your changes in real time 
+* 1 page website styled with CSS
+* Use 1 page website from module 1 and expand to 5 pages with links, embedded media, navigation bar, footer and a new style 
+* Include a web font on your page 
+* Add metadata to your page in HTML
+* A Styled, but static version draft of passion project (no expectation of interactivity that will come from JavaScript)
+
+### Learning Outcomes
+* Comfortable running local web server 
+* Comfortable creating new webpages with HTML & CSS independently  
+* Comfortable adding basic interaction to websites 
+* Comfort styling websites with CSS
+
+### Supporting Educational Resources
+* The Odin Project
+    * [CSS Foundation](https://www.theodinproject.com/paths/foundations/courses/foundations#css-foundations)
+    * [Flexbox](https://www.theodinproject.com/paths/foundations/courses/foundations#flexbox)
+
+## Module 3: Intro to Javascript
+
+### Duration
+4 weeks
+
+### Facilitation
+1-on-1 Mentorship sessions
+
+### Learning Topics
+* Reading other people’s HTML & CSS (exercise)
+* Intro to the Javascript console 
+* Intro to interactive Javascript 
+* Using Javascript libraries 
+* Make a simple game (Tic-Tac-Toe or Rock Paper Scissors or Coin Toss)  with your mentor
+
+### Key Mentee Deliverables 
+* Reproduce the style of your  favorite website homepage from scratch in HTML & CSS
+* Add basic Javascript to add interactive components like a modal (pop-up) or animations 
+* Add at least 1 Javascript library to your page to add functionality 
+* Bonus: deploy your website to Github pages 
+* Functional initial release of passion project.
+
+### Learning Outcomes
+* Using Chrome developer tools to inspect pages 
+* How to log information in the console 
+* How to use buttons and modals for basic interaction 
+* How to use HTML, CSS & Javascript together 
+* Reinforce Github skills 
+
+### Supporting Educational Resources
+* The Odin Project
+    * [JavaScript Basics](https://www.theodinproject.com/paths/foundations/courses/foundations#javascript-basics)
+    * Bonus - [The Back End](https://www.theodinproject.com/paths/foundations/courses/foundations#the-back-end)
+    * [Conclusion](https://www.theodinproject.com/paths/foundations/courses/foundations#conclusion)
+
+#### Footnotes
```

Comment:
I don’t see any items that actually use these footnote markers in Phase 2 (looks like they were pulled over from phase 1)

Reply:
Looks like same wording is in all 3 phases